### PR TITLE
Adding support for plaintext optional custom email body

### PR DESCRIPTION
**Pull request by alimoeeny**

### CODE REVIEW

File: Pod/Classes/BugShaker.swift [hunk 1]
```diff
@@ -92,7 +95,7 @@ extension UIViewController: MFMailComposeViewControllerDelegate {
   }
   
   /**
-   Present the user with a mail compose view with the recipient(s) & subject line
+   Present the user with a mail compose view with the recipient(s) & subject line & body
```
Comment (dtrenz) [inline Pod/Classes/BugShaker.swift]: Maybe this should be:
`...the recipient(s), subject line, and body`

File: Pod/Classes/BugShaker.swift [hunk 2]
```diff
@@ -14,19 +14,22 @@ public class BugShaker {
...
     static var toRecipients: [String]?
     static var subject: String?
+    static var body: String?
   }
   
   // MARK: - Configuration
   
   /**
-  Set bug report email recipient(s) and custom subject line.
+  Set bug report email recipient(s) and custom subject line and body.
```
Comment (dtrenz) [inline Pod/Classes/BugShaker.swift]: Should probably be:
`... email recipient(s), custom subject line, and body.`

File: Example/BugShaker/AppDelegate.swift [hunk 3]
```diff
@@ -20,7 +20,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     *  Configure BugShaker with an array of email recipients (required)
     *  and an optional custom subject line to use for all bug reports.
     */
-    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report")
+
+    let iOSVersion = UIDevice.currentDevice().systemVersion
+    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report", body: "Hi Developers, \n I am running your app on \(iOSVersion ?? "unknown") and wanted to thank you.\n I am also reporting a bug where ... ")
```
Comment (dtrenz) [inline Example/BugShaker/AppDelegate.swift]: I think just a short & simple text intro for the body would work for the example. The iOS version seems to go a bit beyond scope of this PR. Also, the lengthy body text in the example makes the example a little harder to read.

I will be adding device info (i.e. OS version) as a standard component in these reports, very soon.

**Comment by dtrenz** [comment]
I like making the default body customizable. Thanks!

**Comment by alimoeeny** [comment]
Thanks @dtrenz  I changed the comments and simplified the example. But feel free to change things or let me know if you want me to change them.

File: Pod/Classes/BugShaker.swift [hunk 4]
```diff
@@ -14,19 +14,22 @@ public class BugShaker {
...
   /**
-  Set bug report email recipient(s) and custom subject line.
+  Set bug report email recipient(s), custom subject line and body.
   
   - parameter toRecipients: List of email addresses to which the report will be sent.
   - parameter subject:      Custom subject line to use for the report email.
+  - parameter body:         Custom email body (plain text).
   */
-  public class func configure(to toRecipients: [String]!, subject: String?) {
+    public class func configure(to toRecipients: [String]!, subject: String?, body: String?) {
```
Comment (dtrenz) [inline Pod/Classes/BugShaker.swift]: Leading whitespace.

**Comment by dtrenz** [comment]
Excellent! If you good just address that one whitespace line comment, I will merge.

**Comment by alimoeeny** [comment]
@dtrenz don't know how the white space got there, I am usually careful about whitespaces.
Fixed now.

**Pull request merged**